examples/outlier_RMSD.txt

```diff
@@ -0,0 +1,4 @@
+# Reference Force Field: SMIRNOFF 
+# Molecule Set Directory: examples 
+  MolName           	GAFF     	GAFF2    	MMFF94   	MMFF94S  	OPLS2005 	OPLS3    
+#
```

Comment:
is this supposed to be blank?

Reply:
Well I left it with the default cutoff of 2.0 angstroms and these two molecules are very simple from alkethoh. I looked in the normal RMSD file and the largest RMSD is around 0.07 angstroms. I could rerun with that as the cutoff and add the command to the readme if that makes this file make more sense.

Reply:
probably yeah, just for example's sake?